nestkernel/nest_time.h

```diff
@@ -492,11 +492,11 @@ class Time
   double
   get_ms() const
   {
-    if ( tics == LIM_POS_INF.tics )
+    if ( tics >= LIM_POS_INF.tics )
```

Comment:
I realize this change is based on a suggestion I made, but I think we can do better here. I checked the `Time` class once more and we have checks in place there that ensure that `LIM_NEG_INF.tics <= tics <= LIM_POS_INF.tics` at all times. We also have a `is_neg_inf()` method; adding a corresponding `is_pos_inf()` method, we can then turn this into
```c++
if ( is_pos_inf() )
{
  return LIM_POS_INF.tics;
}
if ( is_neg_inf() )
{
  return LIM_NEG_INF.tics;
}
```
I think this would be even more readable.